yolox/utils/mlflow_logger.py

```diff
@@ -0,0 +1,373 @@
+#!/usr/bin/env python3
+# Copyright (c) Megvii Inc. All rights reserved.
+# Please read docs/mlflow_integration.md for more details.
+
+import json
```

Comment:
`isort`/ `ruff check --fix` this file(and your update) please

Reply:
![image](https://github.com/Megvii-BaseDetection/YOLOX/assets/16800094/8d45a946-d9f5-4dd9-826d-3a8add78df5e)
 Done